README.md

```diff
@@ -68,34 +68,49 @@ import dame_flame
 df = pd.read_csv("dame_flame/data/sample.csv")
 result = dame_flame.DAME_FLAME.DAME(input_data=df, treatment_column_name="treated", outcome_column_name="outcome", holdout_data=1.0)
 print(result[0])
-#>    x1   x2   x3   x4
-#> 0   0   1    1    *
-#> 1   0   1    1    *
-#> 2   1   0    *    *
-#> 3   1   0    *    *
+#>    x1   x2   x3   x4    weights
+#> 0   0   1    1    *     1
+#> 1   0   1    1    *     1
+#> 2   1   0    *    1     1
+#> 3   1   0    *    1     1
+
+print(result[1])
+#> [[0,1],[2,3]]
 ```
-result is a list, where the first element in the list is of type **Data Frame**. The dataframe contains all of the units that were matched, and the covariates and corresponding values, that it was matched on. The covariates that each unit was not matched on is denoted with a " * " character. The list 'result' will have additional values based on additional optional parameters, detailed in additional documentation below. 
+result is a list, where the first element in the list is of type **Data Frame**. The dataframe contains all of the units that were matched, and the covariates and corresponding values, that it was matched on. The covariates that each unit was not matched on is denoted with a " * " character. This element also includes a column of unit weights which specifies the number of times each unit was matched. The second element is a list of unit IDs belonging to each of the matched groups created by the algorithm. The list 'result' will have additional values based on additional optional parameters, detailed in additional documentation below. 
 
-To find the main matched group of a particular unit after DAME has been run, use the function *mmg_of_unit*
+To find the main matched group of a particular unit or group of units after DAME has been run, use the function *MG*:
 
 ```Python
-mmg = dame_flame.DAME_FLAME.mmg_of_unit(return_df=result[0], unit_id=0, input_data=df)
+mmg = dame_flame.DAME_FLAME.MG(return_df=result, unit_id=0, input_data=df)
```

Comment:
I guess you must have checked with the team to confirm you like the new naming of these functions? I'm fine with whatever personally

Reply:
Oh I actually forgot to ask about the names - I will send a slack message now